### ISSUE TITLE
Question: Can "../.." be prepended to a path?

### ISSUE THREAD

**Issue by FredHappyface** [vemel/handsdown#18]
Thanks for the project, Just have a question if that's ok?

Is it possible to configure handsdown such that instead of prepending blob/main to a path, ../.. can be prepended to a path as below?

```diff
-[[find in source code]](blob/main/ansitoimg/ansirep.py#L104)
+[[find in source code]](../../ansitoimg/ansirep.py#L104)
```

**Comment by vemel** [comment]
Hello! Makes perfect sense to me, so I am going to add this feature tomorrow.

**Comment by FredHappyface** [comment]
Awesome stuff! Thank you very much! :D

**Comment by vemel** [comment]
I have finally released `1.0.0`. You can specify a custom path with `--source-code-path <path>` argument. In your case it is

```bash
handsdown --cleanup --source-code-path '../..'
```

Please let me know if it works for you as expected.

**Comment by FredHappyface** [comment]
Awesome stuff! Works perfectly! Thank you :)
